aten/src/ATen/core/Tensor.cpp

```diff
@@ -40,4 +48,162 @@ std::string Tensor::toString() const {
   return type().toString();
 }
 
+// Define the methods out-of-line when they access fields on AutogradMeta
```

Comment:
Did you changed any of these functions on top of moving them?

Reply:
Yes. I'll provide diffs shortly.